orchestration/dap_orchestration/pipelines.py

```diff
@@ -16,7 +16,8 @@
         "beam_runner": local_beam_runner,
         "refresh_directory": refresh_directory,
         "outfiles_writer": outfiles_writer,
-        "api_token": preconfigure_resource_for_mode(api_token, "local")
+        "api_token": preconfigure_resource_for_mode(api_token, "local"),
+        "io_manager": fs_io_manager
```

Comment:
This allows us to resume a pipeline run from the failed step, when running locally.

Reply:
That will be very helpful!